src/interfaces/TodoData.ts

```diff
@@ -0,0 +1,6 @@
+export interface TodoData {
+  [key: string]: string;
```

Comment:
What does this dynamic `key` holds?

Reply:
I was using handler function to pass down for editing the states. TS was giving error with the type of keys in an object so had to define the datatype of the keys